### ISSUE TITLE
Shutdown error

### ISSUE THREAD

**Issue by pikeas** [just-containers/s6-overlay#76]
I have a base image that uses s6-overlay:

```
FROM ubuntu-debootstrap:15.04

<snip>
ENV S6_LOGGING 0
ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2
COPY s6-overlay-1.14.0.1 /
ENTRYPOINT ["/init"]
```

I then start a container:

```
$ docker run --rm -it aris/base bash
<s6-overlay startup logs>
root@76248caedbb2:/# ls
bin   dev  home  lib    media  opt   root  sbin  sys  usr
boot  etc  init  lib64  mnt    proc  run   srv   tmp  var
```

Now I `ctrl+d` (or use `exit`):
```
root@76248caedbb2:/# exit
bash exited 0
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] syncing disks.
s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

Check out `s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening`. I'm not sure what's causing this?

**Comment by bobelev** [comment]
I have the same issue. Just upgraded from 1.13.0 and got this.

**Comment by glerchundi** [comment]
good catch, it should be fixed in `v1.14.0.2`. please try and tell me something.

**Comment by glerchundi** [comment]
@pikeas @bobelev, as far as I know, this is no longer happening, i'm going to close this one.
